lib/crowbar/client/app/upgrade.rb

```diff
@@ -219,12 +219,12 @@ def backup(component)
           catch_errors(e)
         end
 
-        desc "repocheck ADDON",
-          "Check for existing repositories for an addon (ha|storage)"
+        desc "repocheck COMPONENT",
+          "Check for existing repositories for a component (crowbar|ha|storage)"
```

Comment:
to me it is not confusing at all. so not only cloud, but also sles12sp2 is checked, so that are the repos that crowbar itself needs, so why not call the component `crowbar`?